index.ts

```diff
@@ -301,17 +303,20 @@ collect.set('isRepoPRList', [
 	'https://github.com/sindresorhus/refined-github/pulls?q=is%3Apr+is%3Aclosed',
 ]);
 
-// `issues/fregante` is a list but `issues/1`, `issues/new`, `issues/new/choose`, `issues/templates/edit` aren’t
 export const isRepoIssueList = (url: URL | HTMLAnchorElement | Location = location): boolean =>
-	Boolean(getRepo(url)?.path.startsWith('issues')) &&
-	!/^issues\/(\d+|new|templates)($|\/)/.test(getRepo(url)?.path!);
+	// `issues/fregante` is a list but `issues/1`, `issues/new`, `issues/new/choose`, `issues/templates/edit` aren’t
+	/^issues(?!\/(\d+|new|templates)($|\/))/.test(getRepo(url)?.path!) ||
+	Boolean(getRepo(url)?.path.startsWith('labels/'));
```

Comment:
Is there a clever way not to call `getRepo()` twice? I couldn't find another example of this.